### PR TITLE
Testing grunt

### PR DESCRIPTION
**Pull request by IgorSinyak**

### CODE REVIEW

File: scripts/controllers.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      filtByScale: '24.5',
+      filtByPrice: '10000'  
+    };
+    $scope.shortDescription = function(description) {
+      var text = description.substr(0, 50) + "...";
+      return text;
+    };
+
+    $scope.uncheck = function (event) {
+      if ($scope.checked === event.target.value) {
```
Comment (vitali-usik) [inline scripts/controllers.js]: If the type of $scope.checked is boolean you needn't to write `if ($scope.checked === event.target.value)`. Just `if ($scope.checked)`

File: scripts/controllers.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  .controller('FooterController', ['$scope', function($scope) {
+    $scope.showDropdown = false;
+    $scope.dropdownToggle = function() {
+      if($scope.showDropdown) {
+        $scope.showDropdown = false;
+      } else {
+        $scope.showDropdown = true;
+      }
+    };
+    $scope.dropdownValue = 'Dropdown';
```
Comment (vitali-usik) [inline scripts/controllers.js]: It'd be better if you initialize all values in the top of controller. Then functions with blank line like separator. It'll increase readability of code

File: scripts/controllers.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    };
+  }])
+
+  .controller('FooterController', ['$scope', function($scope) {
+    $scope.showDropdown = false;
+    $scope.dropdownToggle = function() {
+      if($scope.showDropdown) {
+        $scope.showDropdown = false;
+      } else {
+        $scope.showDropdown = true;
```
Comment (vitali-usik) [inline scripts/controllers.js]: Can you simplify to $scope.showDropdown = !$scope.showDropdown ?

File: scss/views/contactus.scss [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+        outline: none;
+      }
+
+      select {
+        @include input-styles;
+        position: relative;
+        height: 50px;
+        width: 200px;
+        -webkit-appearance: none;
+        -moz-appearance: none;
```
Comment (vitali-usik) [inline scss/views/contactus.scss:118]: You can add something like autoprefixer, that can do it on compilation phase

File: views/contactus.html [hunk 5]
```diff
@@ -0,0 +1,269 @@
+<div class="contactus" ng-controller="ContactController">
+  <div class="container">
+    <div class="left-column">
+      <ul class="select-list">
+        <li class="select-item"><a href="#" class="active">Lorem Ipsum</a></li>
```
Comment (vitali-usik) [inline views/contactus.html]: Is that element always has class='active' or it's dynamically changing?

File: views/contactus.html [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+        <li class="select-item"><a href="#">Loremipsum & Dolar Sit Dum</a></li>
+        <li class="select-item"><a href="#">Lorem Ipsum Dolar</a></li>
+        <li class="select-item"><a href="#">Lorem Ipsu</a></li>
+      </ul>
+    </div>
+    <div class="right-column">
+      <form class="contactus-form" name="contact" ng-submit="sendForm()" novalidate>
```
Comment (vitali-usik) [inline views/contactus.html]: You need to add 'data-' prefix for all non-html attrs

File: views/contactus.html [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+        <div class="photo-block input-block">
+          <div class="title-column">
+            <span>Photo</span>  
+          </div>
+          <div class="input-column">
+            <input type="file" id="load-photo" name="load-photo" ng-model="contactForm.photo">
```
Comment (vitali-usik) [inline views/contactus.html]: As above

File: views/contactus.html [hunk 8]
```diff
@@ -0,0 +1,269 @@
...
+            <span>SIT</span>
+          </div>
+          <div class="input-column">
+            <select name="xxx" id="sit-xxx">
+              <option value="" selected="selected">xxx</option>
+              <option value="123">123</option>
+              <option value="223">223</option>
+              <option value="564">564</option>
+              <option value="594">594</option>
+              <option value="375">375</option>
```
Comment (vitali-usik) [inline views/contactus.html]: It'd be better to pass an array of options in ng-repeat

File: views/contactus.html [hunk 9]
```diff
@@ -0,0 +1,269 @@
...
+            </select>
+            <select name="xx" id="sit-xx">
+              <option value="" selected="selected">xx</option>
+              <option value="12">12</option>
+              <option value="22">22</option>
+              <option value="56">56</option>
+              <option value="59">59</option>
+              <option value="37">37</option>
```
Comment (vitali-usik) [inline views/contactus.html]: As above

File: views/contactus.html [hunk 10]
```diff
@@ -0,0 +1,269 @@
...
+            </select>
+            <select name="xxxx" id="sit-xxxx">
+              <option value="" selected="selected">xxxx</option>
+              <option value="1231">1231</option>
+              <option value="2232">2232</option>
+              <option value="5643">5643</option>
+              <option value="5944">5944</option>
+              <option value="3754">3754</option>
```
Comment (vitali-usik) [inline views/contactus.html]: As above

File: views/footer.html [hunk 11]
```diff
@@ -1,19 +1,31 @@
...
-            <a href="">Dropdown</a>
+        <div class="dropdown-select" click-out="showDropdown = false">
+          <div class="select dropdown-block-item" >
+            <a href="" ng-click="dropdownToggle()">{{dropdownValue}}</a>
           </div>
+          <ul class="options-list" ng-class="{'options-list-active': showDropdown}">
+            <li class="option"><a href="" ng-click="selectOption($event)">Option1</a></li>
+            <li class="option"><a href="" ng-click="selectOption($event)">Option2</a></li>
+            <li class="option"><a href="" ng-click="selectOption($event)">Option3</a></li>
+            <li class="option"><a href="" ng-click="selectOption($event)">Option4</a></li>
```
Comment (vitali-usik) [inline views/footer.html]: As above

**Review by vitali-usik — CHANGES_REQUESTED** [review]

File: scripts/controllers.js (hunk 1 above)
Comment (IgorSinyak) [reply scripts/controllers.js]: ok, agree with you

File: scripts/controllers.js (hunk 2 above)
Comment (IgorSinyak) [reply scripts/controllers.js]: ok, agree, while studying angular, hard to control code style, i`ll correct code

File: scss/views/contactus.scss (hunk 4 above)
Comment (IgorSinyak) [reply scss/views/contactus.scss:118]: in grunt?

File: views/contactus.html (hunk 5 above)
Comment (IgorSinyak) [reply views/contactus.html]: it dynamically changes, i added it for look style in browser

File: views/contactus.html (hunk 6 above)
Comment (IgorSinyak) [reply views/contactus.html]: 'data-ng-submit', and so for all ng directives?

File: scripts/controllers.js (hunk 3 above)
Comment (IgorSinyak) [reply scripts/controllers.js]: "onclick = function() { $scope.showDropdown = !$scope.showDropdown}", right?

File: scripts/controllers.js (hunk 1 above)
Comment (IgorSinyak) [reply scripts/controllers.js]: this function don`t need here at all, i forgot to delete it =)

File: scripts/controllers.js (hunk 3 above)
Comment (vitali-usik) [reply scripts/controllers.js]: yep

File: scss/views/contactus.scss (hunk 4 above)
Comment (vitali-usik) [reply scss/views/contactus.scss:118]: yep

File: views/contactus.html (hunk 5 above)
Comment (vitali-usik) [reply views/contactus.html]: You can use angular variable like { 'active': isActive() }. Don't you?

File: views/contactus.html (hunk 6 above)
Comment (vitali-usik) [reply views/contactus.html]: yep

**Pull request merged**